guides/security/security_clearance_guidance.md

```diff
@@ -1,15 +1,46 @@
 # Guidance for Security Cleared Staff
 
-In this section we have provided some additional information for team members who are undertaking, or intend to undertake the higher level background checks for security clearance.
+## General information
+There are 3 routes for requesting Security Clearance (SC) and the route taken depends on the project.
```

Comment:
do we want this to be public?

Reply:
I thought it might be helpful for people looking to join but have re-worded